### PR TITLE
home and account cleanup

### PR DESCRIPTION
**Pull request by 0xBigBoss**

### CODE REVIEW

File: packages/app/components/HomeTopNav.tsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import { useThemeSetting } from '@tamagui/next-theme'
 import { IconArrowLeft, IconHamburger, IconQr, IconSendLogo } from 'app/components/icons'
 import { usePathname } from 'app/utils/usePathname'
 import { useRouter } from 'solito/router'
 
-export function HomeTopNav({ header, subheader }: { header: string; subheader?: string }) {
+export function HomeTopNav({
+  header,
+  subheader,
+}: { header: string; subheader?: string; submenuHeader?: string }) {
```
Comment (youngkidwarrior) [inline packages/app/components/HomeTopNav.tsx:11]: Do we need submenuHeader. The design just change the `header` prop 

[e.g for Personal Information]( https://www.figma.com/file/d9CZyOeYO42w4yOpeJbHES/Send-Designs-1.0?type=design&node-id=1331-1516&mode=design&t=lWlAjKjHmXlHOQ7A-4.)

File: packages/app/features/account/AccountTopNav.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  const handleHomeBottomSheet = () => {
+    setNavParam(nav ? undefined : 'home', { webBehavior: 'replace' })
+  }
+
+  const handleSettingsBottomSheet = () => {
+    setNavParam(nav ? undefined : 'settings', { webBehavior: 'replace' })
+  }
+
+  const handleBack = () => {
```
Comment (youngkidwarrior) [inline packages/app/features/account/AccountTopNav.tsx:28]: Nice, I like the assert. 

side note, this is definitely hacky and I probably shouldn't have written it like this. Fine for now, but in the future should probably us the solito router + Next `URL` to do things like this

File: packages/app/features/home/layout.web.tsx [hunk 3]
```diff
@@ -2,16 +2,31 @@ import { YStack, ScrollView } from '@my/ui'
...
+  header: string
+  subheader: string
+}
+
 export function HomeLayout({
   children,
   header = '',
   subheader = '',
-}: { children: React.ReactNode; header?: string; subheader?: string }) {
+  TopNav,
```
Comment (youngkidwarrior) [inline packages/app/features/home/layout.web.tsx:14]: nice

File: packages/app/features/home/screen.tsx [hunk 4]
```diff
@@ -66,13 +66,15 @@ export function HomeScreen() {
...
                         </Paragraph>
-                        <XStack style={{ color: 'white' }} gap={'$2.5'}>
+                        <XStack style={{ color: 'white' }} gap={'$2.5'} mt={'$3'}>
                           {totalBalance === undefined ? (
                             <Spinner size={'large'} />
                           ) : (
                             <Paragraph
                               color={'$color12'}
-                              size={'$14'}
+                              // fontFamily={'$mono'}
```
Comment (youngkidwarrior) [inline packages/app/features/home/screen.tsx:75]: did you mean to comment these out?

File: packages/ui/src/tamagui.config.ts [hunk 5]
```diff
@@ -22,7 +22,7 @@ const conf = {
   fonts: {
     heading: headingFont,
     body: bodyFont,
-    mono: monoFont,
+    mono: process.env.NODE_ENV === 'test' && monoFont === undefined ? bodyFont : monoFont, // monoFont doesn't work in jest tests for some reason
```
Comment (youngkidwarrior) [inline packages/ui/src/tamagui.config.ts:25]: Interesting. I'll look into this, but is should all be importing the same way

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
Looks good. Couple things and NITs 

Current colors used in settings pages make light mode unusable. Handle in future PR

File: packages/app/features/account/settings/layout.web.tsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { Container, Separator, XStack, YStack, useMedia } from '@my/ui'
-import { settingsLinks } from 'app/components/settings/SettingsBottomSheet'
-import { SettingsNavLink } from 'app/components/settings/SettingsNavLink'
+import { settingsLinks } from './SettingsBottomSheet'
```
[the remark this answers is not in the document]
Comment (0xBigBoss) [reply packages/app/features/account/settings/layout.web.tsx:2]: Tough to say. These links won't be web specific I bet and will need to co-exist with native. Ideally we'd move it all to nav component or something.

File: packages/app/features/home/screen.tsx (hunk 4 above)
Comment (0xBigBoss) [reply packages/app/features/home/screen.tsx:75]: no good eye

File: packages/ui/src/tamagui.config.ts (hunk 5 above)
Comment (0xBigBoss) [reply packages/ui/src/tamagui.config.ts:25]: Don't worry about, pretty sure it's a tamagui thing. Pretty sure `@tamagui/core/native-test` is one big hack.

File: packages/app/components/HomeTopNav.tsx (hunk 1 above)
Comment (0xBigBoss) [reply packages/app/components/HomeTopNav.tsx]: No. this shuold be dropped now.

**Review by 0xBigBoss — COMMENTED** [review]
> Looks good. Couple things and NITs 

> Current colors used in settings pages make light mode unusable. Handle in future PR 

yeah there's a lot of improvement to made here.

**Pull request merged**